internal/core/metadata/rest_addressable_test.go

```diff
@@ -152,7 +157,11 @@ func TestAddAddressable(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			rr := httptest.NewRecorder()
-			restAddAddressable(rr, tt.request, logger.NewMockClient(), tt.dbMock)
+			restAddAddressable(
+				rr,
+				tt.request,
+				logger.NewMockClient(),
+				tt.dbMock, errorconcept.NewErrorHandler(logger.NewMockClient()))
```

Comment:
These two parameters should be on separate lines to adhere to the folding pattern.